src/material-examples/chips-drag-and-drop/chips-drag-drop-example.html

```diff
@@ -0,0 +1,10 @@
+<mat-chip-list 
```

Comment:
We use 2 spaces for indentation, but it looks like this file is using 4. Also add a newline at the end of the file.